### ISSUE TITLE
Possible bug in core userCanUseNamespace

### ISSUE THREAD

**Issue by foot** [weaveworks/weave-gitops-enterprise#907]
After #905 was merged into `main`, and after logging in as `wego-admin` to demo-01. Navigating to Applications failed, citing `wego-admin` not having permissions to list helm-releases in the `default` namespace.

`wego-admin` doesn't have _much_ access to default, it had a cluster-role-bindings for:
- https://github.com/weaveworks/weave-gitops-enterprise/blob/2bc52aa2ab0b57df436861c85532967e4b62484b/charts/mccp/templates/rbac/user_roles.yaml#L52-L59
- https://github.com/weaveworks/weave-gitops-enterprise/blob/2bc52aa2ab0b57df436861c85532967e4b62484b/charts/mccp/templates/rbac/user_roles.yaml#L75-L80

With the new bit on the end (`events`) which seems to have broken things...
